packages/notifi-frontend-client/lib/client/NotifiFrontendClient.ts

```diff
@@ -1070,6 +1071,13 @@ export class NotifiFrontendClient {
       (it) => it?.name === name,
     );
     if (existing !== undefined) {
+      // Web push targets are different from the other targets in the way that it isn't managed in the Frontend card.
+      // Instead, web push targets are managed by the service worker in the client's PWA.
+      // If webPushTargetIds is undefined, FE client will retain the existing web push targets in the target group.
+      if (!webPushTargetIds) {
+        webPushTargetIds = existing.webPushTargets?.map<string>(target => target?.id ?? '') ?? []
```

Comment:
Shall we do it the same manner as other targets? 
```
ensureWebPush(this._service, webPushId),
```
Or do we have any concern?

Reply:
Yea that works with me

Reply:
Just wanna double check when can we expect to have the update for this? @kai-notifi